sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/planning/impatient.sc

```diff
@@ -0,0 +1 @@
+1+1;
```

Comment:
Hello Josh: Sorry, it is unintentional. I have deleted the new file and push again.